src/command/list/human.rs

```diff
@@ -58,46 +114,272 @@ fn display_all(
+            .initial_indent(INDENTATION)
+            .subsequent_indent(INDENTATION)
+            .fill(
+                &host_packages
+                    .iter()
+                    .map(list_package)
+                    .collect::<Vec<String>>()
+                    .join("\n"),
+            );
+        format!("⚡️ Tool `{}` available from:\n\n{}", tool, versions)
+    }
+}
+
+/// Format a sets binaries inside `Toolchain::Package`s without detail information
+fn display_tool_sets(packages: &[Package]) -> String {
+    format!(
+        "{}",
+        &packages
+            .iter()
+            .map(display_tool_set)
+            .collect::<Vec<String>>()
+            .join("\n")
+    )
+}
+/// Format a set binaries inside a single `Toolchain::Package` without detail information
+fn display_tool_set(package: &Package) -> String {
+    match package {
+        Package::Default { tools, .. } | Package::Project { tools, .. } => {
+            let tools = match tools.len() {
+                0 => String::from(""),
+                _ => format!("{}", tools.join(", ")),
+            };
+            let width = text_width().unwrap_or(0);
+
+            Wrapper::new(width)
+                .initial_indent(INDENTATION)
+                .subsequent_indent(INDENTATION)
+                .fill(&format!("{}{}", tools, list_package_source(package)))
+        }
+        Package::Fetched(..) => String::new(),
+    }
+}
+
+/// List a set versions of `Toolchain::Node`s.
+fn list_runtimes(runtimes: &[Node]) -> String {
+    if runtimes.is_empty() {
+        String::new()
+    } else {
+        let width = text_width().unwrap_or(0);
+        Wrapper::new(width)
+            .initial_indent(INDENTATION)
+            .subsequent_indent(INDENTATION)
+            .fill(
+                &runtimes
+                    .iter()
+                    .map(|runtime| list_runtime(&runtime))
+                    .collect::<Vec<String>>()
+                    .join("\n"),
+            )
+    }
+}
+
+/// List a single version of `Toolchain::Node`.
+fn list_runtime(runtime: &Node) -> String {
+    format!("v{}{}", runtime.version, runtime.source)
+}
+
+/// List all the versions of `Toolchain::PackageManager`s.
+fn list_package_managers(package_managers: &[PackageManager]) -> String {
+    if package_managers.is_empty() {
+        String::new()
+    } else {
+        let width = text_width().unwrap_or(0);
+        Wrapper::new(width)
+            .initial_indent(INDENTATION)
+            .subsequent_indent(INDENTATION)
+            .fill(
+                &package_managers
+                    .iter()
+                    .map(list_package_manager)
+                    .collect::<Vec<String>>()
+                    .join("\n"),
+            )
+    }
+}
+
+/// List a the version of `Toolchain::PackageManager`.
+fn list_package_manager(package_manager: &PackageManager) -> String {
+    format!("v{}{}", package_manager.version, package_manager.source)
+}
```

Comment:
This is a personal style, but I generally try to avoid having functions whose names are only off by one character (`list_package_manager` and `list_package_managers`). Also, `list` generally implies that there are more than one, could we make this name more descriptive and unique? Maybe something like `format_single_package_manager`?